.concourse/scripts/t-cargo-build.sh

```diff
@@ -42,7 +42,9 @@ fi
 if [ ! -f ".git/resource/changed_files" ] || grep -v '^.concourse\|^Dockerfile\|^scripts/cli' ".git/resource/changed_files" || [ ! -f "target/release/polymesh" ]; then
     rm -f target/release/polymesh
     sed -i -e "s/^version = .*$/version = \"$VERSION\"/" Cargo.toml
+    cargo sweep -s -r
     cargo build --release
+    cargo sweep -f -r
```

Comment:
This will clean everything except for the current release. I don't think we want that because often times we work with different dependency versions in parallel. i.e. I might be working on version 0.1 while someone else on 0.2 and we'd want both of them to remain cached.

Perhaps we can use something like `-t 30` that deletes everything older than 30 days.

//Same comment applies to other cargo sweep usage

Reply:
@maxsam4 Cache artifact reaping needs to be way more aggressive than `-t 30`.  I've had to purge caches nearly every other day over the past two weeks.  The largest cache culprits are consistently rocksdb and the polymesh runtime, which to my eye get recompiled on nearly every build anyway and take up 99% of the build time.  

The other issue is that timestamps are not preserved in the cache.  I therefore don't think that using `-t 30` or even `-t 3` will do anything.

An option might be to delete any file larger than 200MB after building.  That will at least get rid of `liblibrocksdb_sys-*` and `polymesh_runtime_tests-*` which alone responsible for about 80% of the cache growth.

Reply:
>An option might be to delete any file larger than 200MB after building. That will at least get rid of liblibrocksdb_sys-* and polymesh_runtime_tests-* which alone responsible for about 80% of the cache growth.

They take the most time to compile as well. Time taken is directly proportional to size here so you can delete whatever you want, really.

Perhaps we can trigger it based on size? i.e. cleanup if total cache size > 10 GB. It's not perfect but it should give us some breathing room.

Reply:
I am ok with anything between 3-10 GB for the limit. ps Since we aren't using cargo-sweep anymore, you can remove it from the images if you haven't already.